pkgs/development/tools/misc/itmdump/default.nix

```
@@ -0,0 +1,28 @@
+{ stdenv, fetchFromGitHub, rustPlatform, pkg-config }:
+
+rustPlatform.buildRustPackage rec {
+  version = "0.3.1";
+  pname = "itm";
```

Swap order (since it's more canonical).

```suggestion
  pname = "itm";
  version = "0.3.1";
```